### ISSUE TITLE
Implement `to_latest: true`

### ISSUE THREAD

**Issue by mkmik** [google-github-actions/deploy-cloudrun#466]
### TL;DR

When using `no_traffic: true` once, it's hard to set the traffic 

[Docs](https://cloud.google.com/sdk/gcloud/reference/run/deploy#FLAGS) for `--no-traffic` state:

> [After a deployment with this flag the LATEST revision will not receive traffic on future deployments](https://cloud.google.com/sdk/gcloud/reference/run/deploy#FLAGS:~:text=After%20a%20deployment%20with%20this%20flag%20the%20LATEST%20revision%20will%20not%20receive%20traffic%20on%20future%20deployments)

This can be worked around by adding another step that does:

```yaml
run: gcloud run services update-traffic ${{service}} --region ${{region}} --to-latest
```

The current situation is suboptimal because:

1. the user needs to duplicate the service and region parameters (or factor them out in variables) that are also used by the main deploy step
2. it's hard to discover because the deploy-cloudrun page doesn't mention this issue of using `--no-traffic`

### Detailed design

Expose a `to_latest` value that if `true` will cause the action to call `update-traffic` with the `--to-latest` flag, similarly how presence of the `revision_traffic` and `tag_traffic` values already do

**Comment by wshobson** [comment]
Could we please get a status update on the implementation progress of this detailed design?

**Comment by sethvargo** [comment]
```yaml
revision_traffic: 'LATEST=100'
```
